content/22.vaccines.md

```diff
@@ -255,9 +258,84 @@ Some common successful whole virus vaccine include targeting influenza viruses,
 
 #### SinoVac's CoronaVac
 
-<!--https://www.thelancet.com/journals/laninf/article/PIIS1473-3099(20)30843-4/fulltext-->
-
-#### Sinopharm (2 candidates)
+The CoronaVac vaccine is being developed by SinoVac, Beijing-based biopharmaceutical company.
+The vaccine is using an inactivate whole virus with the addition of an aluminum adjuvant [@doi:10.3389/fimmu.2020.585354].
+The vaccine is currently in Phase III clinical trials --> check status.
+
+Pre-clinical trials were performed using BALB/c mice and rhesus macaques [@doi:10.1126/science.abc1932].
+The SARS-CoV2 strains used in this trial isolated from 11 hospitalized patients (5 from China, 3 from Italy, 1 from the UK, 1 from Spain).
```

Comment:
```suggestion
The SARS-CoV-2 strains used in this trial isolated from 11 hospitalized patients (5 from China, 3 from Italy, 1 from the UK, 1 from Spain).
```

This only adds up to 10 patients.  Is there one missing?

Reply:
Ah thanks for checking. I am missing one!